src/module.c

```diff
@@ -6479,14 +6479,24 @@ const char *RM_CallReplyProto(RedisModuleCallReply *reply, size_t *len) {
  * be executed before the next RM_Call is made.  By calling RM_FlushExecutionUnit between RM_Call operations, the
  * module will get those semantics.
  *
+ * Currently, the RedisModuleCtx is not used and only NULL should be passed
```

Comment:
i wouldn't document that it isn't used, and i'd certainly not validate that it **is** NULL.
the contrary, even if we're not using it, let's validate that it is **not** NULL.
then, i think it's clear and there's no need to document anything about it.